source/remediation_runbooks/scripts/CreateCloudTrailMultiRegionTrail_createloggingbucket.py

```diff
@@ -29,14 +34,18 @@ def create_logging_bucket(event, _):
     return {"logging_bucket": bucket_name}
 
 
-def create_bucket(s3, bucket_name, aws_region):
+def create_bucket(s3: S3Client, bucket_name: str, aws_region: str) -> str:
     try:
-        kwargs = {"Bucket": bucket_name, "ACL": "private"}
+        kwargs = {
+            "Bucket": bucket_name,
+            "ACL": "private",
+            "ObjectOwnership": "ObjectWriter",
```

Comment:
is this needed here? isn't `ACL:private` the default config and in that case `ObjectOwenrship` might not be needed to be set to `ObjectWriter`

Reply:
The actions are broken into steps for some reason - we later put an ACL that allows write from LogDelivery. See `put_bucket_acl`.

Reply:
👍